atlp-rwanda/script-heroes-frontend
# #174797473 Allow manager to reject a request

#### What does this PR do?
This PR displays the manager's requests that are associated with him

#### Description of Task to be completed?
- Creating a model for warning
- Creating actions and redux reducer for handling state and backend responses 
- Updating the state of requests

#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend`
- Switch your working dir `cd script-heroes-frontend`
- checkout the branch `git checkout ft-reject-request-#174797473 `
- install all dependencies` yarn install`
- start the app `yarn run dev`
- got to` /request/manager` to display all requests 
- Click on the decision button and click reject
> After loading you should see the state of the request changed to rejected

#### Any background context you want to provide?
- Make sure you are logged in on the app
- Make sure you have completed your profile info
- Make sure you have `MANAGER` as a role on the app
- Make sure that there is some request from someone who has you as his/her line-manager

#### What are the relevant pivotal tracker stories?
[#174797473](https://www.pivotaltracker.com/story/show/174797473)

#### Screenshots (if appropriate)
